app/src/main/java/com/example/androidacademyhw/details/ActorsAdapter.kt

```diff
@@ -24,8 +26,13 @@ class ActorsAdapter : ListAdapter<Actor, ActorsAdapter.ViewHolder>(ActorDiffCall
         RecyclerView.ViewHolder(binding.root) {
         fun bind(item: Actor) {
             with(binding) {
+
+                Glide.with(root.context)
+                    .load(item.picture)
+                    .placeholder(R.drawable.ic_launcher_foreground)
+                    .into(imageActor)
```

Comment:
What about the error handling?

Reply:
Would you be so kind to clarify if you mean try catch or just to add .error(R.drawable.123) in builder?

Reply:
The second one

Reply:
Done